core/src/test/scala/integration/kafka/network/DynamicNumNetworkThreadsTest.scala

```diff
@@ -44,19 +48,29 @@ class DynamicNumNetworkThreadsTest extends BaseRequestTest {
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
     super.setUp(testInfo)
-    TestUtils.createTopic(zkClient, "test", brokerCount, brokerCount, servers)
+
+//    TestUtils.createTopic(zkClientOrNull, "test", brokerCount, brokerCount, brokers)
+    admin = TestUtils.createAdminClient(brokers, ListenerName.forSecurityProtocol(SecurityProtocol.PLAINTEXT))
     assertEquals(2, getNumNetworkThreads(internal))
+    TestUtils.createTopicWithAdmin(admin, "test", brokers, controllerServers)
     assertEquals(Defaults.NumNetworkThreads, getNumNetworkThreads(external))
   }
+  @AfterEach
+  override def tearDown(): Unit = {
+//    TestUtils.shutdownServers(brokers)
```

Comment:
Sorry for the confusion, yes it is. At first I thought I need to shutdown the brokers as I try to make an admin broker before each but just the **admin.close()** is enough so that comment is not needed.